### PR TITLE
add rails api

### PR DESCRIPTION
**Pull request by HectorTorresE**
* Create a new Rails API app called 'backend'.
 * Initialize your project with Git.
 * Make sure that your project has a Postgres database set up.
 * Create a table for storing your messages and create 5 different greetings.
 * Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route).
 * Create a new React app called 'hello-react-front-end'
 * Initialize your project with Git.
 * Create the App component with react-router.
 * Create the Greeting component that will display a greeting. Set it up as a route in your App component.
 * Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
 * Display the random greeting in your Greeting component.
 * In the README file for the Rails API, add a link to your React app.
 * When submitting for a review - use a link to the pull request in your Rails API.
 * In the repo for the frontend part - include a PR with changes.
 
 Frontend repo --> https://github.com/HectorTorresE/frontend

run backend using command ```rails server -p 4000```
run frontend using ```npm start```

### CODE REVIEW

File: app/controllers/api/v1/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
+      def index
+        render json: Greeting.offset(Random.rand(0..4)).first.message
+      end
+    end
+  end
+end
```
Comment (mateo951) [inline app/controllers/api/v1/greetings_controller.rb:9]: Well done! API endpoint for returning a random greeting was created 🔥

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @HectorTorresE, 

**🔥Highlights🔥**
- API 📝
- `Greeting` component 🐐
- Frontend uses the URL to the endpoint from the Rails API 🔥
- Working app ✔

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Please provide a link to the Rails API (backend app) in your README file (frontend) 📝
- Kindly add routes using react-router to your App component as it is part of the requirements 📝
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by HectorTorresE** [comment]
Thanks for the review the change have been implemented in the frontend repo

**Review by descholar-ceo — APPROVED** [review]
Hi @HectorTorresE , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**